Kernel/Arch/aarch64/init.cpp

```diff
@@ -86,11 +89,15 @@ Atomic<Graphics::Console*> g_boot_console;
 
 extern "C" [[noreturn]] void init()
 {
+    g_in_early_boot = true;
```

Comment:
who turns it off?

Reply:
Nobody yet, we don’t exit early boot at the moment.